source/Halibut.Tests/Builders/ServiceFactoryBuilder.cs

```diff
@@ -27,8 +21,9 @@ public ServiceFactoryBuilder WithService<TContract>(Func<TContract> factoryFunc)
             }
             // Convention verification may throw, but that just means we're probably going to use
```

Comment:
@nathanwoctopusdeploy I think this comment is redundant now. Remove?

Reply:
Yeh seems redundant

Reply:
Deleted!